<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoSelectionPage1</class>
 <widget class="QWidget" name="VideoSelectionPage1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="title">
     <property name="styleSheet">
      <string notr="true">font: 38pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Titolo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="text">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>testo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="warning_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
background-color: rgb(255, 190, 190);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="warning_icon">
        <property name="text">
         <string/>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="warning_text">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;Rubik Light&quot;;
color: rgb(255, 0, 0)</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="error_text">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(255,0,0);</string>
     </property>
     <property name="text">
      <string>Per favore fornisci il tuo consenso per iniziare lo studio</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkbox">
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Rubik Light&quot;;</string>
     </property>
     <property name="text">
      <string>Ho letto e compreso le informazioni riguardo questo studio e sono disposto a partecipare</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="exit_button">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
font: 15pt &quot;Rubik Light&quot;;
color: rgb(255,255,255);
background-color: rgb(0, 64, 130);</string>
       </property>
       <property name="text">
        <string>Esci</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ready_button">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
font: 15pt &quot;Rubik Light&quot;;
color: rgb(255,255,255);
background-color: rgb(0, 64, 130);</string>
       </property>
       <property name="text">
        <string>Pulsante</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="bottom_text">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>testo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
